src/components/BrowseSkill/BrowseSkill.js

```diff
@@ -299,6 +299,31 @@ export default class BrowseSkill extends React.Component {
       },
     });
   };
+
+  loadTopUsed = () => {
+    let url;
+    url =
+      urls.API_URL +
+      '/cms/getSkillList.json?group=All&applyFilter=true&filter_name=descending&filter_type=usage&count=10';
```

Comment:
You also need to send a duration parameter along with this API, like duration=7 for most used skills in this week.

Reply:
@AnupKumarPanwar The PR deals with showing the most used skills in general. For a week's data, user can do that from the sidebar.